### PR TITLE
Implemented notify me only under delegatee notification settings

### PR DESCRIPTION
**Pull request by saitejasFyle**
While implementing, found an existing issue which is, the selected mode of notification under delegatee notification dropdown was not getting populated once we revisit the notifications page after saving the changes.

This has also been taken care in this change by cachebusting and caching the latest changes.

### CODE REVIEW

File: src/app/core/models/org_user_settings.model.ts [hunk 1]
```diff
@@ -91,7 +91,8 @@ export interface NotificationSettings {
   email: Email;
   push: Push;
   whatsapp: Whatsapp;
-  notify_only_delegatee: boolean;
+  notify_delegatee: boolean;
```
Comment (Dimple16) [inline src/app/core/models/org_user_settings.model.ts:94]: are these present every time?

**Review by Dimple16 — APPROVED** [review]

File: src/app/core/models/org_user_settings.model.ts (hunk 1 above)
Comment (saitejasFyle) [reply src/app/core/models/org_user_settings.model.ts:94]: Yeah

**Review by Julias0 — CHANGES_REQUESTED** [review]

File: src/app/fyle/notifications/notifications.page.ts [hunk 2]
```diff
@@ -261,18 +282,21 @@ export class NotificationsPage implements OnInit {
...
       emailEvents: false,
       pushEvents: false,
     };
-
     this.orgUserSettings$ = this.orgUserSettingsService.get();
 
     let notifyOption;
     this.updateDelegateeSubscription().subscribe((option) => {
       notifyOption = option;
+      if (this.notificationForm) {
```
[the remark this answers is not in the document]
Comment (saitejasFyle) [reply src/app/fyle/notifications/notifications.page.ts:296]: Yeah whenever we subscribe to updateDelegateSubscription(), if a form group instance (notificationForm) is already present, we have to populate the current notifyOption by setting the value to that form control. So to check if the formGroup is undefined or not, I have added the if condition.

File: src/app/fyle/my-profile/my-profile.page.ts [hunk 3]
```diff
@@ -121,9 +121,11 @@ export class MyProfilePage implements OnInit {
     this.ROUTER_API_ENDPOINT = environment.ROUTER_API_ENDPOINT;
   }
 
-  reset() {
+  async reset() {
     this.eou$ = from(this.authService.getEou());
-    const orgUserSettings$ = this.offlineService.getOrgUserSettings().pipe(shareReplay(1));
+    const orgUserSettings$ = await this.offlineService
```
[the remark this answers is not in the document]
Comment (saitejasFyle) [reply src/app/fyle/my-profile/my-profile.page.ts]: It's done. And I've raised the PR

File: src/app/fyle/notifications/notifications.page.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Julias0) [reply src/app/fyle/notifications/notifications.page.ts:296]: Lets talk about this on monday

**Review by Julias0 — APPROVED** [review]

**Pull request merged**